src/renderer/pages/toast/index.js

```
@@ -630,6 +631,23 @@ function setupEventListeners() {
   });
+
+  // 특별 명령어 처리 (꽃가루 애니메이션 등)
+  window.toast.onSpecialCommand = function (command) {
+    console.log('Special command received:', command);
+
+    if (command === 'confetti' || command === '꽃가루') {
+      // 꽃가루 애니메이션 실행
+      showStatus('🎉 Let it go!', 'success');
+      window.confetti.start({
+        duration: 5,  // 5초 동안 실행
+        density: 100  // 꽃가루 밀도
```

ESLint

[error] Missing trailing comma.

(comma-dangle)